lib/modules/asset/collections/assetsMappings.ts

```diff
@@ -25,7 +25,10 @@ export const assetsMappings: CollectionMappings = {
         date: { type: "date" },
       },
     },
-
+    softTenant: {
```

Comment:
So there could be only 1 `softTenant`?

Reply:
No, but it's currently implemented like that and change this need a migration of database so it's breaking if we change to pluralize now